### DISCUSSION TITLE
Linux server

### DISCUSSION THREAD

**Discussion by abysalvinthomas** [apohl79/audiogridder#359] [Ideas]
If AG release a Linux server that host windows plugins through wine....it will be so useful.
linux gives more stability..malware free than windows.
thank you

**Comment by CaBuffalo** [comment]
The plugins will have to support Linux. I don't think AG can do that.

Good idea though.

**Reply by abysalvinthomas** [reply]
If AG can scan .wine folder , I think it is possible

**Reply by rcrath** [reply]
there are linux compiled vsts, quite a few. running them would require compiling the linux server.  Sshouldn't wine run the windows server? Wine should not need a special linux version of audiogridder. I'm going to try that, but still trying to get the linux server to compile.

**Comment by abysalvinthomas** [comment]
I tried AG server with wine. But not working perfectly.
The plugin side is not showing the scanned interfaces of plug-ins

Thank you

**Comment by dromer** [comment]
You could run Carla as a vst and host your windows plugins inside that.

**Comment by Scout064** [comment]
Well, I currently run AudioGridder under the following Setup:
Debian 10 4.0.19-0-rt-amd64 and the latest Wine (12.02.2021) (would need to check which Version exactly)
It runs okay but as soon as I load a "heavy" VST for e.g. GuitarRig 6 I have to increase the Buffer from 0 to at least 4 otherwise the VST does not Load, or the Server hangs.
Even with that, it is pretty sluggish other VST's are running perfectly fine even with no Buffer.

**Comment by abysalvinthomas** [comment]
Wow...its a happy news of hope..!!
If it works fine ...we can build a server under linux Os.
i expecting a native linux AG server..that can scan and host vst/vsti from
wine partition ( installed through wine)
any way ..happy
